tutor/commands/upgrade/common.py

```diff
@@ -66,3 +66,55 @@ def get_mongo_upgrade_parameters(
     mv '/openedx/data/ora2/SET-ME-PLEASE (ex. bucket-name)' /openedx/data/ora2/openedxuploads
 fi
 """
+
+def get_mysql_change_charset_query(
```

Comment:
nit: we should move this util to another file as we are not using this in upgrade anymore.

Reply:
Moved this method to the `tutor/utils.py` file.